definitions/npm/element-ready_v3.x.x/flow_v0.25.x-/test_element-ready.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+import elementReady, {type Options} from 'element-ready';
```

Comment:
Can you group using `describe` / `it` blocks to make it easier for future maintainers?